packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -2,6 +2,30 @@
  * Tabs component
  */
 export class Tabs {
+  /** @private */
+  $module
+
+  /** @private */
+  $tabs
+
+  /** @private */
+  keys = { left: 37, right: 39, up: 38, down: 40 }
+
+  /** @private */
+  jsHiddenClass = 'govuk-tabs__panel--hidden'
+
+  /** @private */
+  changingHash = false
+
+  /** @private */
+  boundTabClick
+
+  /** @private */
+  boundTabKeydown
+
+  /** @private */
+  boundOnHashChange
+
```

Comment:
**question** Noticed while looking at #3958 that this list is missing the `mql` field, which is still first used in `setupResponsiveChecks` a little further. Is it intentional, or just that is was missed among all the other fields? 😊

Reply:
@romaricpascal Nice spot, yeah it was missed. Added it 👍